src/style.css

```diff
@@ -0,0 +1,88 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+* {
+  box-sizing: border-box;
+  font-family: Avenir, sans-serif;
```

Comment:
Don't use inside `*` selector anything except `box-sizing`.
Use `font-family` rule inside `html` selector.

Reply:
After that you should add `font-family: inherit` for `.input` and its placeholder selectors.